.github/scripts/print_latest_commits.py

```diff
@@ -92,7 +89,11 @@ def isGreen(commit: str, results: Dict[str, Any]) -> Tuple[bool, str]:
             return (False, workflowName + " checks were not successful")
 
     if not all(regex.values()):
```

Comment:
In this case, the `if not all(regex.values())` is no longer necessary, since you can just check if missing_workflows is nonempty